Use pkg-config to find freetype

Polynomial-C opened the pull request:
As of freetype-2.9.1 the freetype-config script has been deprecated and
is no longer shipped by default.

strk commented:
Travis doesn't like your change, please check that

Polynomial-C commented:
Travis seems to be happy now.

Polynomial-C commented:
Ping. Anything else I need to do before this can be merged?

strk commented:
The `./configure --help` command gives:
```
  --enable-freetype       Enable freetype support (default: enabled)
```

At the end of my `./configure` run I get this report:
```
  Freetype library disabled
```

And no other trace of looking for freetype. Is it _off_ by default instead of _enabled_ as stated by --help ?

strk commented:
Travis also shows a discrepancy:
https://travis-ci.org/libming/libming/builds/382649627#L317 -- freetype is found
https://travis-ci.org/libming/libming/builds/382649627#L413 -- freetype is said to be disabled

Polynomial-C commented:
Nice catch. Should be fixed now.

Polynomial-C commented:
Yes, it is enabled by default but this is nothing I have changed with my patch. It was already enabled by default before this patch.
I have changed the description to `--disable-freetype`.

strk commented:
thanks :)
